### PR TITLE
fix: there is different space between folders and chats

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
- Issue: https://github.com/epam/ai-dial-chat/issues/342

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Deployment status: success

File: src/components/Sidebar/BetweenFoldersLine.tsx [hunk 1]
```diff
@@ -79,7 +79,7 @@ export const BetweenFoldersLine = ({
       onDragEnter={highlightDrop}
       onDragLeave={removeHighlight}
       ref={dragDropElement}
-      className={classNames('h-1', isDraggingOver && highlightColorBg)}
+      className={classNames('h-0.5', isDraggingOver && highlightColorBg)}
```
Comment (IlyaBondar) [inline src/components/Sidebar/BetweenFoldersLine.tsx]: on Drag&Drop this area is narrow
image
Need to decide what we expect

**Comment by IlyaBondar** [comment]
/deploy-review
>Deployment status: success

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Deployment status: success

**Pull request closed without merging**